### ISSUE TITLE
How do I change the type of a property in a DOM interface?

### ISSUE THREAD

**Issue by trusktr** [microsoft/TypeScript-DOM-lib-generator#1364]
Specifically this: https://github.com/microsoft/TypeScript/pull/50031

I'm not clear on how to make that change here.

**Comment by saschanaz** [comment]
Yeah, unfortunately supporting this would complicate things. We can't just redefine the field in `.iterable.d.ts`. 😞

**Comment by saschanaz** [comment]
(At this point I wonder we should get a totally separate dom.lib.d.ts for ES2015+ support, rather than augmenting ES5 version of the lib.)
